schemas/test/en/test_benchmark_business.json

```
@@ -5,7 +5,7 @@
     "data_version": "0.0.1",
     "survey_id": "017",
     "form_type": "0070",
-    "theme": "northernireland",
+    "theme": "default",
```

We should set to business anyway.
```suggestion
    "theme": "business",
```